common/src/sql/dbinit.sql

```diff
@@ -86,6 +86,10 @@ CREATE TABLE omicron.public.sled (
     part_number STRING(63) NOT NULL,
     revision INT8 NOT NULL,
 
+    /* CPU & RAM summary for the sled */
+    online_logical_cpus INT8 CHECK (online_logical_cpus BETWEEN 0 AND 4294967295) NOT NULL,
```

Comment:
Yes, though this gets nuanced in the future. See https://github.com/oxidecomputer/omicron/pull/2299#discussion_r1099457140 and the fact that we could have online logical cpus that aren't usable for VM guests in a core-aware scheduling world because you wouldn't have a working twin (but it could be usable for others).